server/chat-plugins/auction.ts

```diff
@@ -503,42 +506,46 @@ export class Auction extends Rooms.SimpleRoomGame {
 		this.state = 'bid';
 		this.highestBid = this.minBid;
 		this.highestBidder = this.nominatingTeam;
-		this.sendMessage(Utils.html`/html <username class="username">${user.name}</username> from team <b>${this.nominatingTeam.name}</b> has nominated <username>${player.name}</username> for auction. Use /bid to place a bid!`);
+		this.sendMessage(Utils.html`/html <username class="username">${user.name}</username> from team <b>${this.nominatingTeam.name}</b> has nominated <username>${player.name}</username> for auction. Use /bid or simply type a number to place a bid!`);
```

Comment:
In most EU countries the decimal separator is comma, e.g. ``4,5``. It would be nice to either support it or mention that you should use a dot here